java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/KinesisProducerConfiguration.java

```diff
@@ -1340,15 +1328,25 @@ public KinesisProducerConfiguration setThreadingModel(ThreadingModel threadingMo
     }
 
     /**
-     * Sets the maximum number of threads that the native process' thread pool will be configured with.
+     * Sets the threading model that the native process will use.
+     * <p>
+     * See {@link #getThreadingModel()} for more information
      *
+     * @param threadingModel the String representation of the threading model to use.
+     * @return this configuration object
+     */
+    public KinesisProducerConfiguration setThreadingModel(String threadingModel) {
```

Comment:
can you add valid strings to both this java doc and https://github.com/awslabs/amazon-kinesis-producer/blob/master/java/amazon-kinesis-producer-sample/default_config.properties for reference